packages/backend/src/App.ts

```diff
@@ -535,6 +535,14 @@ export default class App {
             }) as RequestHandler,
         );
         expressApp.use(Sentry.Handlers.tracingHandler());
+
+        // Set k8s tags for Sentry
+        Sentry.setTags({
+            podName: this.lightdashConfig.k8s.podName,
```

Comment:
can we make tags camel case, they're easier to read in the sentry UI imo. Also we should mention these are from kubernetes: `k8s_pod_name`

Reply:
sounds good!

Reply:
renamed here: https://github.com/lightdash/lightdash/pull/10299/commits/2431a320f626133d2f764ff8dd715f8771853015